locations/spiders/yettel_rs.py

```diff
@@ -0,0 +1,49 @@
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class YettelRSSpider(scrapy.Spider):
+    name = "yettel_rs"
+    item_attributes = {
+        "brand": "Yettel",
+        "brand_wikidata": "Q1780171",
+        "country": "RS",
+    }
+
+    def start_requests(self):
+        url = "https://www.yettel.rs/stores/latlong/"
+        yield scrapy.http.FormRequest(url=url, formdata={"initial": "true"}, method="POST")
+
+    def parse(self, response):
+        for index, store in enumerate(response.json()["data"]["stores"]):
+            item = Feature()
```

Comment:
We can see the fields of Feature [here](https://github.com/alltheplaces/alltheplaces/blob/master/locations/items.py). But the 2 issues are you're using `postal_code`, it should be `postcode`.
```patch
-            item["postal_code"] = store["post_number"]
+            item["postcode"] = store["post_number"]
```

Also we need a ref:
```suggestion
            item["ref"] = store["id"]
```